operator/cmd/manager/main.go

```diff
@@ -142,26 +158,10 @@ func main() {
 		os.Exit(1)
 	}
 
-	skipWebhookEnvVal := os.Getenv("SKIP_VALIDATING_WEBHOOK")
```

Comment:
From a different PR
@jimdickinson : can we keep this as a thing? not to use it by default, but as an escape hatch?

 @respringer: Yeah, I was a bit nervous about removing this as well. I recommend we keep it around at least until we have done one release with the webhook being defaulted.